src/services/firebase/firestore.ts

```diff
@@ -31,6 +33,7 @@ interface FirestoreProps {
   encryptKey?: string;
   memberIds?: string[];
   blackListWords: string[] | null;
+  encryptionOptions: EncryptionOptions | null;
```

Comment:
```suggestion
  encryptionOptions?: EncryptionOptions;
```

Reply:
I still don't get why it loop forever if it is undefined
Another suggestion is that to refactor `useEffect` in `ChatProvider`, you can check my update on file
https://github.com/saigontechnology/rn-firebase-chat/blob/2da64b5e1edf644fd1283071ff6d8b57a0f59410/src/chat/ChatProvider.tsx

Reply:
ok I removed null. Maybe Im wrong in Provider